### DISCUSSION TITLE
Question: Is it possible to control where the Magick.Native-Q16-x64.dll is placed?

### DISCUSSION THREAD

**Discussion by Sicos1977** [dlemstra/Magick.NET#1073] [Help] [answered]
Hi,

Is it somehow through code possible to control where the Magick.Native-Q16-x64.dll is placed?

I see that on my webserver the dll now gets placed in c:\Windows\Temp\Magick.NET.net20.8.4.0\

I like it to be placed in another folder.

**Comment by dlemstra** [comment, marked as answer]
There is a "chapter" about that in the documentation.
